src/main/webapp/WEB-INF/pages/checkout.jsp

```diff
@@ -0,0 +1,119 @@
+            <c:set var="product" value="${item.product}"/>
+            <tr>
+                <td>
+                    <img class="product-tile" src="${product.imageUrl}">
+                </td>
+                <td>
+                    <a href="${servletContextPath}/products/${product.id}">${product.description}</a>
+                </td>
+                <td class="quantity">
+                    <fmt:formatNumber value="${item.quantity}" var="quantity"/>
+                        ${item.quantity}
+                </td>
+                <td class="price">
+                    <a href="#" onmouseover="showHid(${loop.index})" onmouseleave="showHid(${loop.index})">
+                        <fmt:formatNumber value="${product.price}" type="currency"
+                                          currencySymbol="${product.currency.symbol}"/>
+                    </a>
+                    <div id="${loop.index}" class="popup" style="display: none">
+                        <tags:popup product="${product}"/>
+                    </div>
+                </td>
+            </tr>
+        </c:forEach>
+        <tr>
+            <td>Subtotal</td>
+            <td></td>
+            <td class="quantity">
+                    ${order.totalQuantity}
+            </td>
+            <td class="price">
+                <fmt:formatNumber value="${order.subtotal}" type="currency"
+                                  currencySymbol="${order.currency.symbol}"/>
+            </td>
+        </tr>
+        <tr>
+            <td>Delivery</td>
+            <td></td>
+            <td></td>
+            <td class="price">
+                <fmt:formatNumber value="${order.deliveryCost}" type="currency"
+                                  currencySymbol="${order.currency.symbol}"/>
+            </td>
+        </tr>
+        <tr>
+            <td>Total</td>
+            <td></td>
+            <td></td>
+            <td class="price">
+                <fmt:formatNumber value="${order.totalPrice}" type="currency"
+                                  currencySymbol="${order.currency.symbol}"/>
+            </td>
+        </tr>
+    </table>
+    <h2>Your details</h2>
+    <table>
+    <table>
+        <tags:orderFormRow name="firstName" label="First name" order="${order}" errors="${errors}" type="text" defaultValue="Ваше имя"/>
+        <tags:orderFormRow name="lastName" label="Last name" order="${order}" errors="${errors}" type="text" defaultValue="Ваша фамилия"/>
+        <tags:orderFormRow name="phone" label="Phone" order="${order}" errors="${errors}" type="text" defaultValue="+375(xx)xxxxxxx"/>
+        <tags:orderFormRow name="deliveryDate" label="Delivery date" order="${order}" errors="${errors}" type="date" defaultValue=""/>
+        <tags:orderFormRow name="deliveryAddress" label="Delivery address" order="${order}" errors="${errors}"
+                           type="text" defaultValue="г. Город ул.Ваша улица, дом"/>
```

Comment:
let's leave default value only for phone
because for phone it's impossible to save your +375(xx)xxxxxxx due to validation
but for other fields we can save address, name and surname without filling values, just with the default ones